app/views/decidim/vocdoni/admin/steps/_danger_zone.html.erb

```diff
@@ -1,6 +1,9 @@
+<%# i18n-tasks-use t('decidim.vocdoni.admin.steps.danger_zone.messages.created') %>
+<%# i18n-tasks-use t('decidim.vocdoni.admin.steps.danger_zone.messages.paused') %>
+<%# i18n-tasks-use t('decidim.vocdoni.admin.steps.danger_zone.messages.vote') %>
```

Comment:
I see that we have lots of entries into the `i18n-tasks.yml` file. Maybe we can just add these there?